docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -744,6 +744,7 @@
               timeout: "{{ boot_wait_time }}"
             register: waiting_after_reboot
             ignore_unreachable: true
+            ignore_errors: true
```

Comment:
Errors: It can ignore errors when the task is failed, except those syntax errors, undefined variable errors, connection failures, execution issues.
This occurs if Ansible can reach the host, but the task itself encounters an issue. This could happen if the host is reachable but not fully ready to respond, or if a required port isn’t open yet.